libs/effects/src/lib/to-props.operator.ts

```diff
@@ -0,0 +1,11 @@
+import { OperatorFunction } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Action } from './actions.types';
+
+export function toProps<T extends Action>(): OperatorFunction<
```

Comment:
If this is the old toPayload, do we need toPayload?

Reply:
I've added one. It works only if actions include `payload` property.